### PR TITLE
Fix violations of Style/AndOr and reenable it.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Mix of auto-generated corrections via "bundle exec rubocop --only Style/AndOr --auto-correct" and manual corrections.

### CODE REVIEW

File: dashboard/app/controllers/dynamic_config_controller.rb [hunk 1]
```diff
@@ -30,8 +30,7 @@ def gatekeeper_delete
 
     params.require(:feature)
     feature = params[:feature]
-    where = JSON.load(params[:where]) or {}
-    where = {} if where.nil?
+    where = JSON.load(params[:where]) || {}
```
Comment (ashercodeorg) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: Note this change was a manual change, please review more carefully.

File: dashboard/test/ui/runner.rb [hunk 2]
```diff
@@ -209,13 +209,13 @@ def format_duration(total_seconds)
...
-  if $options.browser and browser['browser'] and $options.browser.casecmp(browser['browser']) != 0
+  if $options.browser && browser['browser'] && $options.browser.casecmp(browser['browser']) != 0
     next
   end
-  if $options.os_version and browser['os_version'] and $options.os_version.casecmp(browser['os_version']) != 0
+  if $options.os_version && browser['os_version'] && $options.os_version.casecmp(browser['os_version']) != 0
     next
   end
-  if $options.browser_version and browser['browser_version'] and $options.browser_version.casecmp(browser['browser_version']) != 0
+  if $options.browser_version && browser['browser_version'] && $options.browser_version.casecmp(browser['browser_version']) != 0
```
Comment (ashercodeorg) [inline dashboard/test/ui/runner.rb:218]: Note these changes (in this file) were manual changes, please review more carefully.

File: dashboard/app/controllers/dynamic_config_controller.rb (hunk 1 above)
Comment (laurelfan) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: does it have test coverage? :)

Comment (ashercodeorg) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: $ ls -l dashboard/test/controllers/dynamic_config_controller_test.rb
ls: dashboard/test/controllers/dynamic_config_controller_test.rb: No such file or directory

:(

**Comment by davidsbailey** [comment]
LGTM

File: dashboard/app/controllers/dynamic_config_controller.rb (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: Though I'm not going to block submission of this PR on test coverage, I will try to get around to writing some simple tests for this controller.

Comment (joshlory) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: Pivotal item?

Comment (ashercodeorg) [inline dashboard/app/controllers/dynamic_config_controller.rb:33]: https://www.pivotaltracker.com/story/show/113967915

**Pull request merged**